src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -1104,6 +1104,7 @@ public static void insertConfigIntoConfigFile(RemoteRobot remoteRobot, String fi
                 keyboard.enter();
 
                 keyboard.enterText(configNameSnippet);
+                TestUtils.sleepAndIgnoreException(5);
```

Comment:
@turkeylurkey,  After checking the build results, I found that only one Windows build failed due to the `diagnostic not appearing in the server.env issue`. Apart from that, the other builds, including Mac and Linux, were successful. Therefore, the probability of failure due to the diagnostic not appearing in the `server.env` issue is very low. I believe this fix is good, and it should be fine to get green builds now, as the issue is not reproducible through manual testing.

Reply:
Thank you for the comment.